documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -58,266 +64,245 @@ Run the example codes below in a `mongo` session to:
-db.setProfilingLevel(2, {slowms: 0})
-```
 
-!!! note alert alert-primary ""
-    If you have already [added a service](#add-service), you should remove it and re-add it after changing the profiling level.
+## Enable MongoDB profiling for Query Analytics (QAN)
 
-## Add service
+To use PMM QAN, you must turn on MongoDB's [profiling feature]. By default, profiling is turned off as it can adversely affect the performance of the database server.
 
-When you have configured your database server, you can add a MongoDB service with the user interface or on the command line.
+Choose one of the following methods to enable profiling:
 
-!!! caution alert alert-warning "Important"
-    To monitor MongoDB sharded clusters, PMM requires access to all cluster components. Make sure to add all config servers, shards, and mongos. Otherwise, PMM will not be able to correctly collect metrics and populate dashboards.
+=== "In MongoDB configuration file (Recommended)"
+    This method ensures your settings persist across server restarts and system reboots. It's the recommended approach for production environments:
+    {.power-number}
+    
+    1. Edit the configuration file (usually `/etc/mongod.conf`).
 
-### With the user interface
+    2. Create or add this to the `operationProfiling` section. ([Read more][MONGODB_CONFIG_OP_PROF].)
 
-To add a service with the UI:
-{.power-number}
+        ```yml
+        operationProfiling:
+          mode: all
+          slowOpThresholdMs: 200
+          rateLimit: 100 # (Only available with Percona Server for MongoDB.)
+        ```
 
-1. Select **PMM Configuration > Add Service > MongoDB**.
+        !!! caution alert alert-warning "Important"
+            This is a [YAML] file. Make sure to pay attention to indentation, as it is crucial for correct parsing.
 
-2. Enter or select values for the fields.
+    3. Restart the `mongod` service using the appropriate command for your system. For example, for `systemd`:
 
-3. Click **Add service**.
+        ```sh
+        systemctl restart mongod
+        ```
 
-![!](../../../images/PMM_Add_Instance_MongoDB.jpg)
+=== "On CLI"
+    Use this method when starting the MongoDB server manually:
 
-### On the command line
+    ```sh
+    mongod --dbpath=DATABASEDIR --profile 2 --slowms 200 --rateLimit 100
+    ```
 
-Use `pmm-admin` to add the database server as a service using one of these example commands.
+    - `--dbpath`: The path to database files (usually `/var/lib/mongo`).
+    - `--profile`: The MongoDB profiling level. A value of `2` tells the server to collect profiling data for *all* operations. To lower the load on the server, use a value of `1` to only record slow operations.
+    - `--slowms`: An operation is classified as *slow* if it runs for longer than this number of milliseconds.
+    - `--rateLimit`: (Only available with Percona Server for MongoDB.) The sample rate of profiled queries. A value of `100` means sample every 100th fast query. ([Read more][PSMDB_RATELIMIT])
```

Comment:
link

Reply:
link to this, I guess, https://docs.percona.com/percona-server-for-mongodb/8.0/rate-limit.html?h=profiling

Reply:
links are defined at the bottom of the doc